app/BibReference.php

```diff
@@ -3,9 +3,9 @@
 namespace App;
 
 use Illuminate\Database\Eloquent\Model;
-use RenanBr\BibTexParser\Listener as Listener;
-use RenanBr\BibTexParser\Parser as Parser;
-use RenanBr\BibTexParser\ParseException as ParseException;
+use RenanBr\BibTexParser\Listener;
+use RenanBr\BibTexParser\Parser;
+use RenanBr\BibTexParser\ParseException;
```

Comment:
### Review of line 8 from app/BibReference.php

- Hello world.
- **Testing.**
- Looks good
- Was “_as ParseException_” something like an alias! @andrechalom ? If so, did you stop using it or did you find a way to use without the explicit declaration?

Reply:
Yes, the `use ... as ...` format creates an alias to the class. However, these particular lines had absolutely no effect, as the result of `RenanBr\BibTexParser\ParseException` is already to expose a class named `ParseException`. 

A similar line, but which creates a real alias, may be found here:
https://github.com/opendatabio/opendatabio/blob/6251ef721f5bab9465b344523f0f40ddedddc756/app/ExternalAPIs.php#L4

Check that in the ExternalAPIs, we use a class named Guzzle (ie, line 29). This name is an alias to the "Client" class in the "GuzzleHttp" namespace. But I thought that having a class named "Client" is very confusing...